source/2020-09-25-the-ember-times-issue-164.md

```diff
@@ -88,15 +88,36 @@ The blog also discusses using [`keyForAttribute`](https://api.emberjs.com/ember-
 <add blurb and emoji to "SOME-INTRO-HERE">
 
 ---
+## [Blog Post: Async Data and Autotracking in Ember Octane ✨](https://v5.chriskrycho.com/journal/async-data-and-autotracking-in-ember-octane/)
```

Comment:
```suggestion

## [Blog Post: Async Data and Autotracking in Ember Octane ✨](https://v5.chriskrycho.com/journal/async-data-and-autotracking-in-ember-octane/)
```

adding a blank line about the `##` will fix markdownlint

//

@chrisrng thanks for the awesome writeup and @ijlee2 for the review! 💯 

i'm going to start a delayed writeup on chris's mixin post so there will be a lot of chris content here!